src/pybamm/config.py

```diff
@@ -118,7 +210,7 @@ def generate():
         return
 
     # Ask the user if they want to opt in to telemetry
-    opt_in = ask_user_opt_in()
+    opt_in = ask_user_opt_in(timeout=10)
```

Comment:
Default is already 10
```suggestion
    opt_in = ask_user_opt_in()
```